native/0003-Add-jni-and-android-interface.patch

```diff
@@ -23,22 +21,22 @@ index 730e51520ee2d..cc19e7d3317fc 100644
    public org.chromium.net.CronetEngine$Builder setStoragePath(java.lang.String);
    public org.chromium.net.CronetEngine$Builder setLibraryLoader(org.chromium.net.CronetEngine$Builder$LibraryLoader);
    public org.chromium.net.CronetEngine$Builder enableQuic(boolean);
-@@ -217,6 +218,7 @@ public abstract class org.chromium.net.ICronetEngineBuilder {
+@@ -218,6 +219,7 @@ public abstract class org.chromium.net.ICronetEngineBuilder {
    public abstract org.chromium.net.ICronetEngineBuilder setStoragePath(java.lang.String);
    public abstract org.chromium.net.ICronetEngineBuilder setUserAgent(java.lang.String);
    public abstract java.lang.String getDefaultUserAgent();
 +  public abstract org.chromium.net.ICronetEngineBuilder setEnvoyUrl(java.lang.String);
    public abstract org.chromium.net.ExperimentalCronetEngine build();
    public org.chromium.net.ICronetEngineBuilder enableNetworkQualityEstimator(boolean);
    public org.chromium.net.ICronetEngineBuilder setThreadPriority(int);
-@@ -401,4 +403,4 @@ public abstract class org.chromium.net.UrlResponseInfo {
-   public abstract java.lang.String getProxyServer();
-   public abstract long getReceivedByteCount();
```

Comment:
this chunk also seemed pretty different.  not sure if that's an issue or not.

Reply:
`components/cronet/android/api.txt` is generated by some of the chromium build tools... changing anything but adding envoy_url and the Stamp value does seem weird. I'll take a look